src/middlewared/middlewared/plugins/vm.py

```diff
@@ -131,18 +131,19 @@ def __init__(self, manager, vm):
 
                 vnc_resolution = device['attributes'].get('vnc_resolution', None)
                 vnc_port = int(device['attributes'].get('vnc_port', 5900 + self.vm['id']))
+                vnc_bind = device['attributes'].get('vnc_bind', '0.0.0.0')
 
```

Comment:
If the IP isn't available anymore the user can't start the VM. I was thinking to back off to 0.0.0.0, but in this case, sounds like a security risk.